src/app/modules/group/pages/managed-groups/managed-groups.component.ts

```diff
@@ -7,7 +10,21 @@ import { Component, OnInit } from '@angular/core';
 })
 export class ManagedGroupsComponent implements OnInit {
 
-  constructor() { }
+  title = 'Groups you manage';
+
+  constructor(
+    private currentContent: CurrentContentService,
+  ) {
+    this.currentContent.setCurrent({
+      type: 'group',
+      breadcrumbs: {
+        category: GroupBreadcrumbCat,
+        path: [{title: this.title, navigateTo: ['groups', 'managed']}],
+        currentPageIdx: 0,
```

Comment:
Just put "Groups you manage" as a category. "Groups > .." is more used for the group hierarchy.

Reply:
Done

Reply:
The category is not correctly selected. As indicated in the struct doc, `currentPageIdx` must be `-1` to select the category.